tap-version-14-specification.md

```diff
@@ -0,0 +1,741 @@
+---
+layout: default
+title: TAP 14 specification
+---
+
+# NAME
+
+TAP14 - The Test Anything Protocol v14
+
+## SYNOPSIS
+
+TAP, the Test Anything Protocol, is a simple text-based interface
+between testing modules a test harness. TAP started life as part of
+the test harness for Perl but now has implementations in C/C++,
+Python, PHP, Perl, JavaScript, MatLab, and probably others by the time
+you read this.
+
+This document describes version 14 of TAP.
+
+## THE TAP14 FORMAT
+
+A TAP stream is a UTF-8 encoded text stream representing a set of
+tests.  It should be parsed line by line.  Lines are separated by
+`\n`.
```

Comment:
I'm speculating here but it might be worth supporting `\r` too due to TAP being so old.